src/components/dropdown/dropdown.js

```diff
@@ -40,10 +41,12 @@ export default class Dropdown extends PureComponent {
     color: undefined,
     align: undefined,
     hoverable: undefined,
+    label: '',
```

Comment:
Please change this to undefined to avoid prop to be shown on the React Dev Tools

Reply:
Oops forgot about that; sorry.